.eslintignore

```diff
@@ -1,5 +1,8 @@
-coverage
-dist
-docs
-tmp
-blueprints/*/files
+!.eslintrc.js
+!.template-lintrc.js
+/coverage
+/dist
+/docs
+/tmp
+/vendor
+/blueprints/*/files
```

Comment:
Good call on anchoring these to the beginning of the path. These directory patterns should technically also have either a trailing slash or /**.